var promise = Promise.resolve('test');

promise.catch(function(err) {
    
});